### PR TITLE
feat(checkbox): use 2020 colors

### PR DESCRIPTION
**Pull request by danethurber**

### CODE REVIEW

File: packages/checkbox/src/css/index.js [hunk 1]
```diff
@@ -20,13 +20,13 @@ export default {
...
-    background: core.colors.gray05,
+    border: `2px solid ${core.colorsTextIcon.lowOnDark}`,
+    background: core.colorsBackgroundDark[1],
     color: core.colors.white
   },
   [`.psds-checkbox__square.psds-theme--${themeNames.light}`]: {
-    background: core.colors.white,
-    borderColor: core.colors.gray04,
+    background: core.colorsBackgroundLight[3],
+    borderColor: core.colorsTextIcon.lowOnLight,
```
Comment (jaketrent) [inline packages/checkbox/src/css/index.js]: matches figma.  seems odd semantic choice from the figma point of view.  why not colorsBorder?

File: packages/checkbox/src/css/index.js [hunk 2]
```diff
@@ -62,26 +62,14 @@ export default {
     right: '-7px',
     bottom: '-7px'
   },
-  '.psds-checkbox__square:focus:before': {
```
Comment (jaketrent) [inline packages/checkbox/src/css/index.js]: what were these?  why can they all go away?

seems related to focus rings.  But in that case, why are there still so many lines of css for focus rings remaining?

File: packages/checkbox/src/css/index.js (hunk 1 above)
Comment (danethurber) [reply packages/checkbox/src/css/index.js]: it's figma the square of the checkbox is considered an icon. i agree is is weird but these colors looks better than the border colors. synced up with brian and this is what we came up with

File: packages/checkbox/src/css/index.js (hunk 2 above)
Comment (danethurber) [reply packages/checkbox/src/css/index.js]: this was missed when we did the focus ring to halo component refactor. there is no :before element to target

**Pull request merged**